docs/src/components/ExampleCode.js

```diff
@@ -61,28 +68,31 @@ export default function ExampleCode({ code, name }: {| code: string, name: strin
             borderStyle="sm"
             display="flex"
             overflow="hidden"
+            maxHeight={!expanded ? CODE_EXAMPLE_HEIGHT : undefined}
             position="relative"
             ref={codeExampleRef}
             rounding={2}
-            {...(expanded ? {} : { maxHeight: CODE_EXAMPLE_HEIGHT, overflow: 'hidden' })}
           >
             <Box
               flex="grow"
               onFocus={() => {
                 setExpanded(true);
               }}
             >
-              {/* We can not pass in an id for LiveEditor which links to the underlying text area */}
-              {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-              <label>
-                <LiveEditor
-                  className="live-editor-pane"
-                  style={{
-                    minHeight: '152px',
-                  }}
-                  padding={16}
-                />
-              </label>
+              <div className={!expanded ? 'LiveEditor__textarea__notExpanded' : undefined}>
+                {/* We can not pass in an id for LiveEditor which links to the underlying text area */}
+                {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+                <label>
+                  <LiveEditor
+                    // Don't remove this className or all the a11y integration tests will fail
```

Comment:
what about adding the context Ayesha just gave? 

`// Don't remove this className or all the a11y integration tests will fail. It's used in commands.js to avoid Cypress testing the accessibility of the code editor due to an issue with it detecting the right background color IIRC.`